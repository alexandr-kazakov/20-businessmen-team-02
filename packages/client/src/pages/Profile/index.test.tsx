import React from 'react'
import { render } from '@testing-library/react'

import ProfilePage from '@/pages/Profile/index'

jest.mock('react-redux', () => ({
  useSelector: () => ({ user: {}, profileView: jest.fn()}),
  useDispatch: jest.fn()
}));

test('should render ProfilePage component', () => {
  const {asFragment} = render(<ProfilePage/>)

  expect(asFragment()).toMatchSnapshot()
})