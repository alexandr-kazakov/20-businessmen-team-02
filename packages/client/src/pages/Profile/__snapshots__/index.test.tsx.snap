// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ProfilePage component 1`] = `
<DocumentFragment>
  <div>
    <div
      class="container"
    >
      <div>
        <label
          tabindex="0"
        >
          <img
            alt=""
            src="https://images.unsplash.com/photo-1633332755192-727a05c4013d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1480&q=80"
          />
          <input
            accept="image/jpeg, image/jpg, image/png"
            type="file"
          />
        </label>
      </div>
      <section>
        <form>
          <ul
            class="clear-list undefined"
          >
            <li>
              <div>
                Почта
              </div>
              <div>
                <input
                  name="email"
                  type="text"
                  value=""
                />
              </div>
            </li>
            <li>
              <div>
                Логин
              </div>
              <div>
                <input
                  name="login"
                  type="text"
                  value=""
                />
              </div>
            </li>
            <li>
              <div>
                Имя
              </div>
              <div>
                <input
                  name="first_name"
                  type="text"
                  value=""
                />
              </div>
            </li>
            <li>
              <div>
                Фамилия
              </div>
              <div>
                <input
                  name="second_name"
                  type="text"
                  value=""
                />
              </div>
            </li>
            <li>
              <div>
                Псевдоним
              </div>
              <div>
                <input
                  name="display_name"
                  type="text"
                  value=""
                />
              </div>
            </li>
            <li>
              <div>
                Телефон
              </div>
              <div>
                <input
                  name="phone"
                  type="text"
                  value=""
                />
              </div>
            </li>
          </ul>
          <button
            class=""
            type="button"
          >
            Изменить
          </button>
        </form>
      </section>
    </div>
  </div>
</DocumentFragment>
`;
